datacenter/dtr/2.2/guides/admin/configure/external-storage/s3.md

```diff
@@ -0,0 +1,116 @@
+generally use the same terminology, though setup may be slightly different.
+
+### About S3
+
+S3 stores data as objects within “buckets” where you read, write, and delete
+objects in that bucket.  All read and write operations will be sent to S3 (or
+your S3-compatible server), ensuring availability and durability of your images.
+
+### Configuring S3 itself
+
+This section deals with creating and configuring bucket policies within AWS; if
+you're using an S3-compatible server you may skip this section.
+
+Prior to configuring DTR you need to make a "bucket" within S3.  Buckets are
+uniquely named containers in which S3 stores files.
+
+You must:
+
+1. Create a bucket within S3, choosing a region which is closest to your DTR
+2. Note the bucket and region name for configuring DTR
+
+You then need to configure authorization for your bucket.  You can choose to use
+an [access and secret key combination](
+http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html) for
+the entirety of DTR (simple, but potentially less secure) or configure an [IAM
+policy for the bucket and DTR](
+http://docs.aws.amazon.com/AmazonS3/latest/dev/example-policies-s3.html) (more
+complex to configure but also more secure, as access is restricted to only the
+bucket).
+
+If using an access key and secret key you should copy these and begin
+configuring your storage settings within DTR
+
+**Creating an IAM policy for your bucket**
+
+You can set a policy through your AWS console to manage permissions for DTR.
+For more information about setting IAM policies using the command line or the
+console, review the AWS [Overview of IAM Policies](
+http://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html) article
+or visit the console Policies page.
+
+The following example describes the minimum permissions set which allows
+Trusted Registry users to access, push, pull, and delete images.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "s3:ListAllMyBuckets",
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": "arn:aws:s3:::<INSERT YOUR BUCKET HERE>"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject"
+            ],
+            "Resource": "arn:aws:s3:::<INSERT YOUR BUCKET HERE>/*"
+        }
+    ]
+}
+
+```
+
+To set a policy through the AWS command line, save the policy into a file,
+for example `TrustedRegistryUserPerms.json`, and pass it to the
+put-user-policy AWS command:
+
+```
+$ aws iam put-user-policy --user-name MyUser --policy-name TrustedRegistryUserPerms --policy-document file://C:\Temp\TrustedRegistryUserPerms.json
```

Comment:
We should be consistent with placeholders.
MyUser -> `<your-user>`
file://c:\Temp -> `file://<file-path>`